src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,46 @@
+.nav {
+  &__list {
+    gap: 20px;
+    display: flex;
+    margin: 0;
+    padding: 0;
+    flex-direction: row;
+    list-style: none;
+  }
+
+  &__link {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    height: 60px;
+    text-decoration: none;
+    font-size: 12px;
+    color: $color-link;
+    font-weight: 500;
+    text-transform: uppercase;
+
+    &:hover {
+      color: $color;
+      transition: color 300ms ease-in-out;
```

Comment:
The `transition` property should be specified not only for the hover state but also for the default state to ensure that the transition is smooth in both directions. You can move the `transition` property to the `.nav__link` class.